$OpenBSD: patch-build_properties,v 1.1.1.1 2011/09/21 13:38:21 jasper Exp $

This needs to be set in order for ant to think the build was configured.

--- build.properties.orig	Mon Aug 29 19:30:43 2011
+++ build.properties	Mon Aug 29 19:30:46 2011
@@ -52,7 +52,7 @@ build.compiler modern
 # Class path for the ObjectWeb utility Ant tasks (version 1.3.2 or higher)
 # See http://forge.objectweb.org/projects/monolog
 
-# objectweb.ant.tasks.path ow_util_ant_tasks.jar
+objectweb.ant.tasks.path ow_util_ant_tasks.jar
 
 ###############################################################################
 # TESTS (PERFORMANCE COMPARISONS WITH BCEL AND SERP)
